### DISCUSSION TITLE
Welcome to time2shine Discussions! | Week 1

### DISCUSSION THREAD

**Discussion by tech-n1c** [SE-TINF22B6/time2shine#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AndresMasis** [comment]
Hi, I write to keep the forum discussion going of week 2. When you post, your Product Owner, SCRUM Master, SCRUM master and all the other important things seen on week 2, I will be very interested. Hope to hear more about your project. Cheers.

**Reply by tech-n1c** [reply]
Thanks for post! I just published our weekly report, so you might be interested in that. But it is not complete yet, some things (e. g. the Frontend stuff will follow until next lecture on Thursday).
